model/riscv_insts_crypto.sail

```diff
@@ -0,0 +1,185 @@
+
+/*
+ * file: riscv_insts_crypto.sail
+ *
+ * This file contains instructions whos definitions do not change whether
```

Comment:
```suggestion
 * This file contains instructions whose definitions do not change whether
```
but maybe also just "independent of XLEN" or something as it's both shorter and more general (thinking of RV128).